.github/workflows/release.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       search_url: 'https://api.telescope.cdot.systems/v1/search'
       feed_discovery_url: 'https://api.telescope.cdot.systems/v1/feed-discovery'
       status_url: 'https://api.telescope.cdot.systems/v1/status'
+      supabase_url: 'http://kong:8000'
```

Comment:
This will need to be the production URL: https://telescope.cdot.systems/v1/supabase